spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -671,22 +672,22 @@ it is still possible to dynamically remove metrics from metric registries at run
 === Metric Registries
 
 The `MetricRegistry` is used to maintain a collection of metrics along with their <<pgm-metadata,metadata>>.
-There is one shared singleton of the `MetricRegistry` per scope (_application_, _base_, and _vendor_).
-When metrics are registered using annotations, the metrics are registered in the _application_ `MetricRegistry` (and thus the _application_ scope).
+There is one shared singleton of the `MetricRegistry` per scope (_application_, _base_, _vendor_ and user-defined scopes).
+When metrics are registered using annotations, the metrics are registered in the _application_ `MetricRegistry` (and thus the _application_ scope) by default if the the annotation's _scope_ parameter does not supply a user-defined scope.
 
-When injected, the `@RegistryType` is used as a qualifier to selectively inject either the `APPLICATION`, `BASE`, or `VENDOR` registry.
-If no qualifier is used, the default `MetricRegistry` returned is the `APPLICATION` registry.
+When injected, the `@RegistryScope` is used to selectively inject either an `application`, `base`, `vendor` or a user-defined scope.
+If _scope_ parameter is used, the default `MetricRegistry` returned is the `application` registry.
 
 Implementations may choose to use a Factory class to produce the injectable `MetricRegistry` bean via CDI. See <<appendix#metric-registry-factory>>. Note: The factory would be an internal class and not exposed to the application.
 
-==== @RegistryType
-The `@RegistryType` can be used to retrieve the `MetricRegistry` for a specific scope.
-The implementation must produce the corresponding `MetricRegistry` specified by the `RegistryType`.
+==== @RegistryScope
+The `@RegistryScope` can be used to retrieve the `MetricRegistry` for a specific scope.
+The implementation must produce the corresponding `MetricRegistry` specified by the `RegistryScope`.
 
 NOTE: The implementor can optionally provide a _read_only_ copy of the `MetricRegistry` for _base_ and _vendor_ scopes.
 
 ==== Application Metric Registry
-The implementation must produce the _application_ `MetricRegistry` when no `RegistryType` is provided (`@Default`) or when the `RegistryType` is `APPLICATION`.
+The implementation must produce the _application_ `MetricRegistry` when no `RegistryScope` is provided (`@Default`) or when the `RegistryScope` is `APPLICATION`. Application defined metrics can also be registered to <<pgm-custome-scope,user-defined scopes>>
```

Comment:
"Application defined" -> "Application-defined"

Also, the cross-reference link seems correct (that's the way the anchor is defined) but do we want to change the spelling in the anchor and the reference? `pgm-custome-scope` (an extra 'e'?) (Obviously the difference is invisible to readers and obviously this is a tiny thing!)

Reply:
Can't explain my twitch reaction to add a `e`  to custom :(